test/restClient.spec.js

```diff
@@ -0,0 +1,228 @@
+const expect = require('chai').expect;
```

Comment:
First line must be "use strict"; because 
```
let aorClient, fakeClient, fakeService;
^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
```

Reply:
Hi @josx @AmrN 

Could you help in https://github.com/feathersjs/docs/issues/925? I'll like use sinon for hooks unit testing. This is a great example but I don't know how to stub 2 or 3 services.

Thanks in advance

Reply:
Sorry about the delay, I didnt use sinon.